<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NeuroSegmentParcellation</class>
 <widget class="qMRMLWidget" name="NeuroSegmentParcellation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>998</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="parameterSetLabel">
     <property name="text">
      <string>Parameter set: </string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="qMRMLNodeComboBox" name="parameterNodeSelector" native="true">
     <property name="toolTip">
      <string>Pick node to store parameter set</string>
     </property>
     <property name="nodeTypes" stdset="0">
      <stringlist>
       <string>vtkMRMLScriptedModuleNode</string>
      </stringlist>
     </property>
     <property name="showHidden" stdset="0">
      <bool>true</bool>
     </property>
     <property name="baseName" stdset="0">
      <string>NeuroSegmentParcellation</string>
     </property>
     <property name="renameEnabled" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="ctkCollapsibleButton" name="parcellationParametersCollapsibleButton" native="true">
     <property name="text" stdset="0">
      <string>Parcellation parameters</string>
     </property>
     <property name="collapsed" stdset="0">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="ctkPathLineEdit" name="queryFilePathEdit" native="true"/>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="loadQueryButton">
        <property name="text">
         <string>Update</string>
        </property>
        <property name="iconSize">
         <size>
          <width>12</width>
          <height>12</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Query file:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="ctkCollapsibleButton" name="inputModelCollapsibleButton" native="true">
     <property name="text" stdset="0">
      <string>Input models</string>
     </property>
     <property name="collapsed" stdset="0">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>White surface:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="qMRMLNodeComboBox" name="origModelSelector" native="true">
        <property name="nodeTypes" stdset="0">
         <stringlist>
          <string>vtkMRMLModelNode</string>
         </stringlist>
        </property>
        <property name="noneEnabled" stdset="0">
         <bool>true</bool>
        </property>
        <property name="addEnabled" stdset="0">
         <bool>false</bool>
        </property>
        <property name="removeEnabled" stdset="0">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Pial surface:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="qMRMLNodeComboBox" name="pialModelSelector" native="true">
        <property name="nodeTypes" stdset="0">
         <stringlist>
          <string>vtkMRMLModelNode</string>
         </stringlist>
        </property>
        <property name="noneEnabled" stdset="0">
         <bool>true</bool>
        </property>
        <property name="addEnabled" stdset="0">
         <bool>false</bool>
        </property>
        <property name="removeEnabled" stdset="0">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Inflated surface:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="qMRMLNodeComboBox" name="inflatedModelSelector" native="true">
        <property name="nodeTypes" stdset="0">
         <stringlist>
          <string>vtkMRMLModelNode</string>
         </stringlist>
        </property>
        <property name="noneEnabled" stdset="0">
         <bool>true</bool>
        </property>
        <property name="addEnabled" stdset="0">
         <bool>false</bool>
        </property>
        <property name="removeEnabled" stdset="0">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="ctkCollapsibleButton" name="importMarkupsCollapsibleButton" native="true">
     <property name="text" stdset="0">
      <string>Import</string>
     </property>
     <property name="collapsed" stdset="0">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_6">
      <item row="0" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Type:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QRadioButton" name="markupRadioButton">
          <property name="text">
           <string>Markup</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="overlayRadioButton">
          <property name="text">
           <string>Overlay</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="qMRMLNodeComboBox" name="importMarkupComboBox" native="true">
          <property name="nodeTypes" stdset="0">
           <stringlist>
            <string>vtkMRMLMarkupsNode</string>
           </stringlist>
          </property>
          <property name="addEnabled" stdset="0">
           <bool>false</bool>
          </property>
          <property name="removeEnabled" stdset="0">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ctkComboBox" name="importOverlayComboBox">
          <property name="defaultText" stdset="0">
           <string>Select an overlay</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Source:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Destination:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="qMRMLNodeComboBox" name="destinationMarkupComboBox" native="true">
          <property name="nodeTypes" stdset="0">
           <stringlist>
            <string>vtkMRMLMarkupsNode</string>
           </stringlist>
          </property>
          <property name="addEnabled" stdset="0">
           <bool>false</bool>
          </property>
          <property name="removeEnabled" stdset="0">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="qMRMLNodeComboBox" name="destinationModelComboBox" native="true">
          <property name="nodeTypes" stdset="0">
           <stringlist>
            <string>vtkMRMLModelNode</string>
           </stringlist>
          </property>
          <property name="addEnabled" stdset="0">
           <bool>false</bool>
          </property>
          <property name="removeEnabled" stdset="0">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="importButton">
        <property name="text">
         <string>Import</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QRadioButton" name="singleOverlayRadioButton">
          <property name="text">
           <string>Single</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="multipleOverlayRadioButton">
          <property name="text">
           <string>Multiple</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="ctkCollapsibleButton" name="inputCurvesCollapsibleButton" native="true">
     <property name="text" stdset="0">
      <string>Input curves</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="ctkCollapsibleButton" name="inputPlanesCollapsibleButton" native="true">
     <property name="text" stdset="0">
      <string>Input planes</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2"/>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="ctkCollapsibleButton" name="outputModelsCollapsibleButton" native="true">
     <property name="text" stdset="0">
      <string>Output models</string>
     </property>
     <layout class="QFormLayout" name="formLayout_4"/>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QPushButton" name="applyButton">
     <property name="text">
      <string>Compute all</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="ctkCollapsibleButton" name="exportSegmentationCollapsibleButton" native="true">
     <property name="text" stdset="0">
      <string>Export to segmentation</string>
     </property>
     <layout class="QFormLayout" name="formLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Segmentation:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="qMRMLNodeComboBox" name="exportSegmentationSelector" native="true">
        <property name="nodeTypes" stdset="0">
         <stringlist>
          <string>vtkMRMLSegmentationNode</string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Output structures:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="ctkCheckableComboBox" name="structureSelector"/>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="exportButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Run the algorithm.</string>
        </property>
        <property name="text">
         <string>Export to segmentation</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="exportLabelButton">
        <property name="text">
         <string>Export to surface</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="ctkCollapsibleButton" name="displayCollapsibleButton" native="true">
     <property name="text" stdset="0">
      <string>Display</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="parcellationViewLayoutButton">
        <property name="text">
         <string>Parcellation view layout</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ctkCollapsibleGroupBox" name="CollapsibleGroupBox_3">
        <property name="title">
         <string>General</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="2" column="3">
          <widget class="QRadioButton" name="labelsRadioButton">
           <property name="toolTip">
            <string>Display the parcellation labels on the input models</string>
           </property>
           <property name="text">
            <string>labels</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QRadioButton" name="curvRadioButton">
           <property name="toolTip">
            <string>Display the curvature overlay on the input models</string>
           </property>
           <property name="text">
            <string>curv</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Scalar overlay:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QRadioButton" name="sulcRadioButton">
           <property name="toolTip">
            <string>Display the sulcal height overlay on the input models</string>
           </property>
           <property name="text">
            <string>sulc</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="labelVisibilityCheckBox">
           <property name="toolTip">
            <string>Show/hide the markups label text</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Label visibility:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="controlPointVisibilityCheckBox">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>Control point visibility:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="ctkCollapsibleGroupBox" name="CollapsibleGroupBox">
        <property name="title">
         <string>3D</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="2" column="0">
          <widget class="QLabel" name="label_11">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Plane intersection visibility:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QCheckBox" name="planeIntersectionCheckBox">
           <property name="toolTip">
            <string>Show/hide the intersection of the plane parcellations with the input white surface</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QCheckBox" name="labelOutlineCheckBox">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_13">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Label outline visibility:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="ctkCollapsibleGroupBox" name="CollapsibleGroupBox_2">
        <property name="title">
         <string>2D</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label_8">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Markups curve visibility:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="origMarkupsCheckBox">
           <property name="text">
            <string>White</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="1" colspan="3">
          <widget class="ctkSliderWidget" name="curveIntersectionScaleSlider" native="true">
           <property name="singleStep" stdset="0">
            <double>0.010000000000000</double>
           </property>
           <property name="pageStep" stdset="0">
            <double>0.100000000000000</double>
           </property>
           <property name="maximum" stdset="0">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QCheckBox" name="lineViewGreenCheckBox">
           <property name="text">
            <string>Green</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_15">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Intersection type glyph:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QCheckBox" name="intersectionViewRedCheckBox">
           <property name="text">
            <string>Red</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QCheckBox" name="intersectionViewGreenCheckBox">
           <property name="text">
            <string>Green</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QCheckBox" name="lineViewYellowCheckBox">
           <property name="text">
            <string>Yellow</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="3">
          <widget class="QComboBox" name="intersectionGlyphComboBox"/>
         </item>
         <item row="0" column="2">
          <widget class="QCheckBox" name="pialMarkupsCheckBox">
           <property name="text">
            <string>Pial</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="lineViewRedCheckBox">
           <property name="text">
            <string>Red</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QCheckBox" name="intersectionViewYellowCheckBox">
           <property name="text">
            <string>Yellow</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Intersection glyph scale:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QCheckBox" name="inflatedMarkupsCheckBox">
           <property name="text">
            <string>Inflated</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Intersection views:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Markups curve views:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCheckableComboBox</class>
   <extends>QComboBox</extends>
   <header>ctkCheckableComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>ctkCollapsibleGroupBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkComboBox</class>
   <extends>QComboBox</extends>
   <header>ctkComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>ctkPathLineEdit</class>
   <extends>QWidget</extends>
   <header>ctkPathLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>NeuroSegmentParcellation</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>parameterNodeSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>28</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>NeuroSegmentParcellation</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>origModelSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>NeuroSegmentParcellation</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>exportSegmentationSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>421</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>NeuroSegmentParcellation</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>pialModelSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>NeuroSegmentParcellation</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>inflatedModelSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>NeuroSegmentParcellation</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>importMarkupComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>NeuroSegmentParcellation</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>destinationMarkupComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>NeuroSegmentParcellation</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>destinationModelComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
